packages/p2p/src/components/my-ads/quick-add-modal.jsx

```diff
@@ -457,9 +430,10 @@ const QuickAddModal = ({ advert }) => {
         <Modal
             className='p2p-my-ads__modal-error'
             has_close_icon={false}
-            height='660px'
+            height={my_ads_store.should_show_add_payment_method ? '660px' : 'auto'}
```

Comment:
Height changed according to design:
<img width="611" alt="Screenshot 2022-05-10 at 2 39 45 PM" src="https://user-images.githubusercontent.com/103016120/167564493-9fe2489d-1c88-4780-8be0-e52d4e877e15.png">

Reply:
![Screenshot 2022-05-10 at 12 36 35 PM](https://user-images.githubusercontent.com/103016120/167573904-04f0ffb1-4fe8-4245-810f-add627fb7cbf.png)